src/services/hubspot/createContactService.ts

```diff
@@ -4,7 +4,7 @@ import { Contact } from "./types";
 
 const createContactService = (
     client: IDeskproClient,
-    data: Record<string, string>
+    data: Record<string, unknown>
```

Comment:
Avoid `unknown`

Reply:
The metadata recieved from the API will still have a standard, otherwise how can you code to that? For this reason, I still think typings should be added and we should avoid `unknown`

Reply:
fixed as `Record<string, string>`